### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by ekimbernow**
Signed-off-by: Eliot Kimber <eliot.kimber@servicenow.com>

### CODE REVIEW

File: src/test/java/org/ditacommunity/xml/grammarpool/TestGrammarPoolUsingSAXParserFactory.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import java.util.Date;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class TestGrammarPoolUsingSAXParserFactory {
+
+    @Test
+    public void getParser() throws Exception {
+        GrammarPoolUsingSAXParserFactory factory = (GrammarPoolUsingSAXParserFactory)GrammarPoolUsingSAXParserFactory.newInstance();
```
Comment (drmacro) [inline src/test/java/org/ditacommunity/xml/grammarpool/TestGrammarPoolUsingSAXParserFactory.java:25]: Need to set up a local XML catalog so the parser can succeed without hard coding to an OT instance somewhere.

File: src/test/java/org/ditacommunity/xml/grammarpool/TestGrammarPoolUsingSAXParserFactory.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import javax.xml.parsers.SAXParser;
+import javax.xml.parsers.SAXParserFactory;
+
+import java.net.URL;
+import java.util.Date;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class TestGrammarPoolUsingSAXParserFactory {
```
Comment (drmacro) [inline src/test/java/org/ditacommunity/xml/grammarpool/TestGrammarPoolUsingSAXParserFactory.java:21]: Would be good to have a test that demonstrates the performance improvement at scale.

**Review by drmacro — APPROVED** [review]

File: src/test/java/org/ditacommunity/xml/grammarpool/TestGrammarPoolUsingSAXParserFactory.java (hunk 2 above)
Comment (ekimbernow) [reply src/test/java/org/ditacommunity/xml/grammarpool/TestGrammarPoolUsingSAXParserFactory.java:21]: I'll get to it in another feature branch.

**Comment by ekimbernow** [comment]
Closing this to sort out the commit signing.

**Pull request closed without merging**